### ISSUE TITLE
[iOS] TypeError: UIImage.imageWithData.async is not a function

### ISSUE THREAD

**Issue by jessorlisa** [NativeScript/NativeScript#8986]
**Environment**
 - CLI: 
`├── nativescript@7.0.11`
 - Cross-platform modules: 
`├── @nativescript/core@7.0.12`
 - iOS Runtime: 
`├── @nativescript/ios@7.0.4`
 - XCode Version: 
`12.1 (12A7403)`
 - Plugins:
`├── @nativescript/angular@10.1.7`

**Describe the bug**
When using async images with base64 sources the following error occurs:
```
TypeError: UIImage.imageWithData.async is not a function
at file: node_modules/@nativescript/core/image-source/index.ios.js:129:0
at new ZoneAwarePromise (file: node_modules/@nativescript/zone-js/zone-nativescript.js:909:0)
at Function.fromBase64 (file: node_modules/@nativescript/core/image-source/index.ios.js:126:0)
at Image._createImageSourceFromSrc (file: node_modules/@nativescript/core/ui/image/image-common.js:53:35)
at Image.[src:setNative] (file: node_modules/@nativescript/core/ui/image/index.ios.js:122:0)
at Image.Property.set [as src] (file: node_modules/@nativescript/core/ui/core/properties/index.js:177:0)
at ViewUtil.setPropertyInternal (file: node_modules/@nativescript/angular/__ivy_ngcc__/fesm2015/nativescript-angular.js:2657:0)
at ViewUtil.setProperty (file: node_modules/@nativescript/angular/__ivy_ngcc__/fesm2015/nativescript-angular.js:2631:0)
at EmulatedRenderer.setProperty (file: node_modules/@nativescript/angular/__ivy_ngcc__/fesm2015/nativescript-angular.js:2825:0)
at elementPropertyInternal (file: node_modules/@angular/core/__ivy_ngcc__/fesm2015/core.js:7855:0)
at \M-I\M-5\M-I\M-5propertyInterpolate1 (file: node_modules/@angular/core/__ivy_ngcc__/fesm2015/core.js:15484:0)
at Module.\M-I\M-5\M-I\M-5propertyInterpolate (file: node_modules/@angular/core/__ivy_ngcc__/fesm2015/core.js:15447:0)
...
```

**To Reproduce**
Add an "Image" tag anywhere with a base64 source:
`<Image loadMode="async" src="{{ base64Src }}"></Image>`

**Expected behavior**
No error, working "Image" tag.

**Comment by saschaarthur** [comment]
This error was catched from ios using the patch in https://github.com/NativeScript/nativescript-angular/issues/2278

**Comment by NathanWalker** [comment]
Thanks for reporting @jessorlisa flagging for 7.1 and we'll get this addressed on master soon.

**Comment by NathanWalker** [comment]
@jessorlisa @saschaarthur this is published with 7.0.13 of core now.

**Comment by saschaarthur** [comment]
@NathanWalker awesome thank you !
